components/resources/wallet_strings.grdp

```diff
@@ -496,7 +499,10 @@
   <message name="IDS_BRAVE_WALLET_NUMBERED_ACCOUNT_NAME" desc="The name with account number.">Account <ph name="ACCOUNT_NUMBER">$1<ex>1</ex></ph></message>
   <message name="IDS_BRAVE_WALLET_SIGN_MESSAGE_UNLOCK_FIRST" desc="Sign message error">Wallet needs to be unlocked first</message>
   <message name="IDS_BRAVE_WALLET_SIGN_MESSAGE_INVALID_ADDRESS" desc="Sign message error"><ph name="ADDRESS">$1<ex>1</ex></ph> is not a valid address</message>
-  <message name="IDS_BRAVE_WALLET_SIGN_TYPED_MESSAGE_CHAIN_ID_MISMATCH" desc="Sign typed message error"><ph name="ADDRESS">chainId $1<ex>1</ex></ph> is not current network</message>
+  <message name="IDS_BRAVE_WALLET_SIGN_MESSAGE_CHAIN_ID_MISMATCH" desc="Sign message error"><ph name="CHAIN_ID">chainId $1<ex>1</ex></ph> is not current network</message>
```

Comment:
Is chainID here meant to be inside the placeholder?

Reply:
It doesn't need to be so it can be translated, I was just doing id renaming for this line for it can be reused.
I can move it out if you want.

Reply:
In translations the entire content of the placeholder gets dropped so it would just be 
&lt;ph name="CHAIN_ID"/&gt; is not current network

I am not sure if the word "chainID" is needed here or not. If you are going to insert it programmatically, then it's probably better to move it inside the &lt;ex&gt; tags.

Reply:
Just as long as you are ok with it being dropped the way it is now 😃

Reply:
<img width="467" alt="Screen Shot 2023-09-08 at 10 04 38" src="https://github.com/brave/brave-core/assets/11330831/af496168-1ca6-4adb-a2a3-041e629d0dd5">
oh it is not for translator, it needs to be in the error message for web developers to see